WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -446,7 +446,7 @@ public static void AddKeyPath(this PSBT psbt, HDFingerprint fp, HdPubKey hdPubKe
 		}
 
 		/// <summary>
-		/// Tries to equip the PSBT with previous transactions with best effort.
+		/// Tries to equip the PSBT with previous transactions with best effort. Always <see cref="AddKeyPath"/> first otherwise the prev tx won't be added.
```

Comment:
```suggestion
		/// Tries to equip the PSBT with previous transactions with best effort. Always <see cref="AddKeyPaths"/> first otherwise the prev tx won't be added.
```